problems/lib/problem2.rb

```diff
@@ -0,0 +1,26 @@
+class Problem2
+  def get_largest_two_numbers(array)
+    raise "Array must have at least two elements." if array.size < 2
+
+    if (array[1] > array[0])
+      max1 = array[1]
+      max2 = array[0]
+    else
+      max1 = array[0]
+      max2 = array[1]
+    end

```

Comment:
Can't we initialize with something else to avoid this big chunk of code and incorporate it in our logic ?